pkg/compactor/compactor.go

```diff
@@ -640,7 +648,8 @@ func (c *Compactor) compactUsers(ctx context.Context) {
 	for _, userID := range users {
 		// Ensure the context has not been canceled (ie. compactor shutdown has been triggered).
 		if ctx.Err() != nil {
-			level.Info(c.logger).Log("msg", "interrupting compaction of user blocks", "err", err)
+			interrupted = true
+			level.Info(c.logger).Log("msg", "interrupting compaction of user blocks")
```

Comment:
While you are at it, can we enhance the log message to include value of `userId`? May come in handy for log diving later.

Reply:
done, updated on both of the "interrupting compaction ..." log statements